electronics/ohms_law.py

```diff
@@ -0,0 +1,32 @@
+# Importing doctest for testing our function
+from doctest import testmod
+
+# Defining function
+
+
+def ohms_law(voltage: float = 0, current: float = 0, resistance: float = 0) -> float:
+
+    """
+    This function calculates the any one
+    of the three ohms_law values voltage,
+    current, resistance, of electronics.
+    >>> ohms_law(voltage=10, resistance=5,current=0 )
+    2.0
+    >>> ohms_law(current=1, resistance=10, voltage=0)
+    10.0
+    """
+    if voltage == 0:
+        result = float(current * resistance)
+        return result
```

Comment:
I think your function should return a dict or even better a namedtuple which provides both the name of the value and the value itself.

Creating `result` on one line and getting rid of it on the next is not helpful.

```suggestion
        return {"voltage": float(current * resistance)}
```